.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+# https://www.docker.com/blog/docker-v2-github-action-is-now-ga/
+# https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
+# https://docs.github.com/en/actions/guides/publishing-docker-images
+
+name: CD - Build and push multi-arch images
+
+on:
+  push:
+    branches:
+      - master
+    tags:
```

Comment:
Not quite.
So far, we differentiate between tagged and master-branch releases.

- Tagged releases get the same tag names on dockerhub
- master-releases get the "latest" tag there.

An option would be to skip the master-releases and use the tag trigger only. and then also publishing as "latest".

Reply:
> An option would be to skip the master-releases and use the tag trigger only. and then also publishing as "latest".

That would be ideal, since that's how we run ncp-updates as well

Reply:
I _was_ wondering why the docker containers were all so old, this looks great! Everyone has a lot on their plates right now, but I hope you'll check this out again after the holiday stress is over 🙂 Looks to me like it's nearly ready to be merged once the branch trigger is removed 👍